wxWidgets/3.1.3: conan.py doesn't compile on the raspberry pi (or other arm architectures)

<!-- 
  Please don't forget to update the issue title.
  Include all applicable information to help us reproduce your problem.
-->

### Package and Environment Details (include every applicable attribute)
  * Package Name/Version: **wxWidgets/3.1.3**
  * Operating System+version: **Raspbian 10 (buster)**
  * Compiler+version: **GCC 8.3.0**
  * Conan version: **conan 1.42.0**
  * Python version: **Python 3.7.3**


### Conan profile (output of `conan profile show default` or `conan profile show <profile>` if custom profile is in use)
```
$ conan profile show default
Configuration for profile default:

[settings]
os=Linux
os_build=Linux
arch=armv7
arch_build=armv7
build_type=Release
[options]
[conf]
[build_requires]
[env]

```


### Steps to reproduce (Include if Applicable)
Attempt to compile audacity as per https://github.com/audacity/audacity/blob/master/BUILDING.md


### Logs (Include/Attach if Applicable)
<details><summary>Click to expand log</summary>
```
$ cmake -G "Unix Makefiles" -Daudacity_use_ffmpeg=loaded ../audacity
...
ERROR: Not able to automatically detect '/usr/bin/cc' version
ERROR: Unable to find a working compiler
WARN: wxwidgets/3.1.3.1-audacity: requirement expat/2.2.9 overridden by your conanfile to expat/2.2.9@audacity/stable
wxwidgets/3.1.3.1-audacity: ERROR: while executing system_requirements(): local variable 'arch_suffix' referenced before assignment
ERROR: Error in system requirements
CMake Error at cmake-proxies/cmake-modules/conan.cmake:631 (message):
Conan install failed='1'
Call Stack (most recent call first):
cmake-proxies/cmake-modules/AudacityDependencies.cmake:233 (conan_cmake_install)
cmake-proxies/cmake-modules/AudacityDependencies.cmake:254 (_conan_install)
CMakeLists.txt:509 (resolve_conan_dependencies)
```
</details>


This is my first time compiling Audacity, which fails during the wxWidgets build. I believe bincrafters is the source for the conan file. The issue can be resolved with checking for arm architectures in `system_requirements()`:
```
elif self.settings.arch == 'armv7':
    arch_suffix = ':armhf'
elif self.settings.arch == 'armv8':
    arch_suffix = ':arm64'
```

The audacity issue is https://github.com/audacity/audacity/issues/1390

I'm not super familiar with conan. And I am not an Audacity contributor. Please forgive me if I'm filing an issue against the wrong project.

--- a/recipes/wxwidgets/3.1.3/conanfile.py
+++ b/recipes/wxwidgets/3.1.3/conanfile.py
@@ -113,6 +113,10 @@ class wxWidgetsConan(ConanFile):
                     arch_suffix = ':i386'
                 elif self.settings.arch == 'x86_64':
                     arch_suffix = ':amd64'
+                elif self.settings.arch == 'armv7':
+                    arch_suffix = ':armhf'
+                elif self.settings.arch == 'armv8':
+                    arch_suffix = ':arm64'
                 packages = ['libx11-dev%s' % arch_suffix,
                             'libgtk2.0-dev%s' % arch_suffix]
                 # TODO : GTK3

